etc/workflows/studio-upload.xml

```diff
@@ -0,0 +1,448 @@
+        <configuration key="target-flavor">presentation/prepared</configuration>
+        <configuration key="target-tags">-archive</configuration>
+        <configuration key="rewrite">false</configuration>
+        <configuration key="audio-muxing-source-flavors">*/?,*/*</configuration>
+      </configurations>
+    </operation>
+
+    <!-- Create preview artifacts -->
+
+    <operation
+        id="include"
+        if="NOT ${skipEditorPreview}"
+        description="Prepare preview versions of the recording">
+      <configurations>
+        <configuration key="workflow-id">partial-preview</configuration>
+      </configurations>
+    </operation>
+
+    <operation
+        id="encode"
+        exception-handler-workflow="partial-error"
+        description="Encoding videos to delivery formats">
+      <configurations>
+        <configuration key="source-flavor">*/prepared</configuration>
+        <configuration key="target-flavor">*/delivery</configuration>
+        <configuration key="target-tags">engage-download,engage-streaming</configuration>
+        <configuration key="encoding-profile">studio.adaptive-parallel.http</configuration>
+      </configurations>
+    </operation>
+
+    <!-- Encode to engage player preview images -->
+
+    <operation
+        id="image"
+        exception-handler-workflow="partial-error"
+        description="Creating player preview image">
+      <configurations>
+        <configuration key="source-flavor">*/prepared</configuration>
+        <configuration key="source-tags"></configuration>
+        <configuration key="target-flavor">*/player+preview</configuration>
+        <configuration key="target-tags">engage-download</configuration>
+        <configuration key="encoding-profile">player-preview.http</configuration>
+        <configuration key="time">1</configuration>
+      </configurations>
+    </operation>
+
+    <!-- Encode to feed cover images -->
+
+    <operation
+        id="image"
+        exception-handler-workflow="partial-error"
+        description="Creating feed cover image">
+      <configurations>
+        <configuration key="source-flavor">*/prepared</configuration>
+        <configuration key="source-tags"></configuration>
+        <configuration key="target-flavor">*/feed+preview</configuration>
+        <configuration key="target-tags">atom,rss</configuration>
+        <configuration key="encoding-profile">feed-cover.http</configuration>
+        <configuration key="time">1</configuration>
+      </configurations>
+    </operation>
+
+    <!-- Create a cover image with the default template -->
+
+    <operation
+        id="tag"
+        description="Removing unneeded presenter-cover from download publication">
+      <configurations>
+        <configuration key="source-flavors">presenter/player+preview</configuration>
+        <configuration key="target-tags">-engage-download</configuration>
+      </configurations>
+    </operation>
```

Comment:
Why do you create the image in the first place if you remove the tag here so it's never being used?

Reply:
Same as before, copy paste issue. But the question is, does the player show the presenter preview image?